extensions/amp-story-captions/0.1/amp-story-captions-presets.css

```diff
@@ -0,0 +1,48 @@
+@import '../../amp-story/1.0/amp-story-shadow-reset.css';
+
+/* default */
+
+.amp-story-captions-default .amp-story-captions-cue-wrapper {
+  font-size: calc(2.5 * var(--story-page-vh, 8px)) !important;
+  font-family: 'Roboto', sans-serif !important;
```

Comment:
Can't you change the font of this one?

Reply:
Nope. No customization of the "default" theme.